api/app/routes/admin/shadow_mode.py

```
@@ -522,6 +529,28 @@ async def create_test_response(
 
     repo = _get_repository()
 
+    # Parse status
+    try:
+        shadow_status = ShadowStatus(status)
+    except ValueError:
+        raise HTTPException(
+            status_code=400,
+            detail=f"Invalid status: {status}. Must be one of: "
+            f"{[s.value for s in ShadowStatus]}",
+        )
```

Ruff (0.14.11)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)